docs/get-started/_index.md

```diff
@@ -1,36 +1,97 @@
 ---
-title: "Get Started"
-linkTitle: "Get Started"
+title: "Learn about Viam"
+linkTitle: "Learn about Viam"
 weight: 5
-type: "docs"
+description: "Viam is a complete software platform for smart machines which provides modular components and services for vision, motion, SLAM, ML, and data management."
 no_list: true
+type: docs
+aliases:
+  - "/getting-started/"
+  - "/getting-started/high-level-overview"
+  - "/product-overviews/"
+  - "/viam/"
+  - "/viam/app.viam.com/"
+imageAlt: "/general/understand.png"
+images: ["/general/understand.png"]
+carouselscript: true
 ---
 
-Learn about the Viam platform by reading our Viam in 3 minutes guide or try it out for yourself:
+Viam is a software platform that makes it easy to work with hardware and software.
 
-{{< cards >}}
-{{% card link="/get-started/viam/" %}}
-{{% card link="/get-started/try-viam/" %}}
-{{% card link="/get-started/quickstarts/control-motor/" %}}
-{{% card link="/get-started/quickstarts/collect-data/" %}}
-{{% card link="/get-started/quickstarts/detect-people/" %}}
-{{% card link="/get-started/quickstarts/write-code/" %}}
-{{% card link="/get-started/quickstarts/drive-rover/" %}}
-{{< /cards >}}
+<div>
+{{< imgproc src="/viam/viam.png" alt="Viam overview" resize="800x" class="aligncenter" >}}
```

Comment:
This diagram is pretty well structured IMO, only feedback for now is for the registry box to say "Use existing resources or create and deploy modules to support additional hardware, algorithms, and integrations" or something that indicates this is more than hardware drivers

Reply:
I like that, thanks